demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -704,6 +734,16 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           logLevel
         ),
       );
+
+      await this.createLogStream(configuration, 'create_browser_event_log_stream');
+      this.meetingEventPOSTLogger = new MeetingSessionPOSTLogger(
+        'SDKEvent',
+        configuration,
+        DemoMeetingApp.LOGGER_BATCH_SIZE,
+        DemoMeetingApp.LOGGER_INTERVAL_MS,
```

Comment:
Should we change the interval at which the SDKEvents will be pushed to Cloudwatch to avoid throttling issues.
LOGGER_INTERVAL_MS is used by for 'SDK'.
https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/cloudwatch_limits_cwl.html

Reply:
That's a good point, but it should be okay for two reasons.
- The previous line will add a small delay between two POST loggers.
- The event logger will send only small number of events, e.g. AudioSelected -> Requested -> Succeeded -> Ended

```
// 1. SDK logger starts here

// 2. Wait for the log stream.
await this.createLogStream(configuration, 'create_browser_event_log_stream');

// 3. After a delay, the event logger starts here
this.meetingEventPOSTLogger = new MeetingSessionPOSTLogger(
```